### DISCUSSION TITLE
Cant login in to Postal Dashboard (Problem)

### DISCUSSION THREAD

**Discussion by CryptosWORLD** [postalserver/postal#1909] [Help with using Postal]
Hi
as every month I update and reboot my VPS so I did and this time.
When I when to Postal Dashboard I notice that I can not login
error:  **"The credentials you've provided are incorrect. Please check and try again."**

I have my username and password

I press: Forgotten your password? and add my user but 2nd 
error: **"No user exists with that e-mail address. Please check and try again."**

how can this happen, any ideas?

**Comment by willpower232** [comment]
How are you running your database? is it dockerised MySQL/Maria?

**Comment by CryptosWORLD** [comment]
yes it is it dockerized and running

**Reply by willpower232** [reply]
have you set up a volume for it so that your databases aren't removed if the container gets stopped?

**Comment by CryptosWORLD** [comment]
how i check there is it this config (it is more than a month i have setup it ..) ?

**Reply by willpower232** [reply]
oh I see the documentation doesn't mention you should have a persistent volume...thats unfortunate. I presume you followed it?
https://docs.postalserver.io/install/prerequisites#mariadb

**Comment by CryptosWORLD** [comment]
I follow the guide yes, persistent volume? I don't remember if I did that, how I can do that?
is a way to access my Postal SMTP from Postal Dashboard?

**Reply by CryptosWORLD** [reply]
what I m missing in the above setup ? 
I m looking for the ultimate Postal setup guide or video... 
Can someone help?

**Reply by willpower232** [reply]
The MySQL parts of [this walkthrough](https://medium.com/geekculture/data-persistence-on-mysql-phpmyadmin-using-docker-in-3-steps-c3760d534e41) mention about using a data directory when starting the MySQL docker container so that the database content is not accidentally deleted if the container is removed. I have updating the documentation on my list of things to do.

**Comment by CryptosWORLD** [comment]
Can an admin update the official doc 
https://docs.postalserver.io/
related to persistent volume ..?

**Comment by CryptosWORLD** [comment]
is the below config correct, related to making the data persistent and not deleted after a system update and reboot?

docker run -d \
   --name postal-mariadb \
   -p 127.0.0.1:3306:3306 \
   --restart always \
   -e MARIADB_DATABASE=postal \
   -e MARIADB_ROOT_PASSWORD=postal \
    -v /var/lib/mysql/data:/var/lib/mysql \
    -v /var/lib/mysql/settings:/etc/mysql \
    -v /var/log/mysql:/var/log/mysql \
    -v /var/lib/mysql/backup:/backup \
   mariadb
